shell/platform/darwin/common/framework/Source/FlutterNSBundleUtils.mm

```diff
@@ -8,6 +8,20 @@
 
 FLUTTER_ASSERT_ARC
 
+static GetFlutterAssetPathFromBundle(NSBundle* bundle) {
```

Comment:
@stuartmorgan Refactored to a static function, PTAL

Reply:
This needs a return type. I'm surprised this compiles.

Reply:
Oh, it doesn't. That's a relief 😄